yodeploy/ipc_logging.py

```diff
@@ -49,14 +53,25 @@ def handle(self):
                 break
 
 
-class ThreadedLogStreamServer(SocketServer.ThreadingMixIn,
-                              SocketServer.UnixStreamServer):
+class ThreadedLogStreamServer(socketserver.ThreadingMixIn,
+                              socketserver.UnixStreamServer):
     '''A Server that will receive all log messages sent over the supplied
     (already open) socket
     '''
     def __init__(self):
-        self.socket, self.remote_socket = socket.socketpair(socket.AF_UNIX,
-                                                            socket.SOCK_STREAM)
+        self.socket, self.remote_socket = socket.socketpair(
+            socket.AF_UNIX, socket.SOCK_STREAM)
+
+        # In python >= 3.4, file descriptors are not inheritable by child
+        # processes by default. We need to be able to pass the fd for
+        # self.remote_socket to child processes via a command line argument
+        # in various scenarios, so here we explicitly configure the fd to allow
+        # for that.
```

Comment:
That comment seems a bit unnecessary. The whole point of this module is for such passing. So if we aren't explaining what's going on, adequately, I think that should be done in a module docstring, not before some py3k-specific implementation detail.